This patch avoids the "missing" script to return an error when
makeinfo (from texinfo package) isn't available on host

--- binutils-2.17.50.0.17.orig/missing	2007-10-17 02:04:41.000000000 +0200
+++ binutils-2.17.50.0.17/missing	2007-10-17 02:05:24.000000000 +0200
@@ -299,7 +299,7 @@
     fi
     # If the file does not exist, the user really needs makeinfo;
     # let's fail without touching anything.
-    test -f $file || exit 1
+    # test -f $file || exit 1
     touch $file
     ;;
 
